app/views/reports/show.html.erb

```diff
@@ -0,0 +1,63 @@
+<h2 class="row">The following item needs review for the following reason: </h2>
+<h5 class="row">Reason: <%= @report.reason %></h5>
+<h5 class="row">Reason: <%= @report.message %></h5>
+
+<%= form_with(model: @item, local: true) do |f| %>
+ 
+  <% if @item.errors.any? %>
+    <div id="error_explanation">
+      <h2>
+        <%= pluralize(@item.errors.count, "error") %> prohibited
+        this article from being saved:
+      </h2>
+      <ul>
+        <% @item.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  
+  <p>
+ 
+  <%= f.label :title %>
+        <%= f.text_field :title, class: 'form-control' %>
+        
+        <%= f.label :author %>
+        <%= f.text_field :author, class: 'form-control' %>
+        
+        <%= f.label :description %>
+        <%= f.text_field :description, class: 'form-control' %>
+        
+        <%= f.label :url, 'Optional URL' %>
+        <%= f.text_field :url, class: 'form-control' %>
+        
+        <%= f.label :category %>
+        <%= f.select :category, ["Book", "Podcast", "Video", "Journal", "Other"], {}, {class: 'form-control'}  %>
+
+        <%= f.hidden_field :report_id, :value => @report.id %>
+ </p> 
+  <p>
+    <%= f.submit "Update Libary Item", class: "btn btn-primary" %>
```

Comment:
I'm confused about why the item edit form needs to be displayed here?

Reply:
I think it would be more convenient for the admin to edit the item right on report page without having to navigate to the specific item's editing page.